### PR TITLE
added responsiveness to the home page

### PR DESCRIPTION
**Pull request by RuthAnn1**
I got the home page responsive to small screen sizes.  I took out the banner img for mobiles since it looked really busy with it added.

### CODE REVIEW

File: src/component/navbar/navbar.module.scss [hunk 1]
```diff
@@ -1,29 +1,81 @@
...
+.line:nth-child(2) {
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.line:nth-child(3) {
+  bottom: 0;
+}
+
+@media (max-width: 768px) {
```
Comment (dinaolmelak) [inline src/component/navbar/navbar.module.scss:62]: I see that the `image` is removed when the width gets shorter. Can you also do the same for the height?

**Review by dinaolmelak — CHANGES_REQUESTED** [review]
Looks Good To me

File: src/component/navbar/navbar.module.scss (hunk 1 above)
Comment (RuthAnn1) [reply src/component/navbar/navbar.module.scss:62]: I believe it is fixed.

**Review by Kendiya242 — COMMENTED** [review]
LGTM

File: src/component/navbar/navbar.module.scss (hunk 1 above)
Comment (dinaolmelak) [reply src/component/navbar/navbar.module.scss:62]: Yes it is fixed

**Review by dinaolmelak — APPROVED** [review]

**Comment by RuthAnn1** [comment]
@dinaolmelak  Want to double check.  I fixed the user page and pushed it.  I don't want to merge it unless it is good to go.

**Comment by dinaolmelak** [comment]
@RuthAnn1 , I see the fix for the `userpage` but it no longer shows anything when I hover over it.

**Comment by RuthAnn1** [comment]
You don't see the edits?

**Comment by dinaolmelak** [comment]
@RuthAnn1 , Also the badge is cut off, I can't see it fully unless I resize to different device

**Comment by RuthAnn1** [comment]
Oh I was talking about removing the login on the userpage. It is not responsive yet

**Comment by dinaolmelak** [comment]
Yeah, I see the login is removed

**Comment by RuthAnn1** [comment]
Am I good to merge this branch?

**Comment by dinaolmelak** [comment]
The buttons on the right seem to be gone too

**Comment by RuthAnn1** [comment]
All the buttons are there on my side

**Comment by dinaolmelak** [comment]
I guess maybe it is because the page is not responsive then.

**Comment by RuthAnn1** [comment]
It could be... Is it ok if I merge the branch?

**Comment by dinaolmelak** [comment]
Yeah, the responsive can be taken care of like the `home` page

**Pull request merged**